runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java

```diff
@@ -469,4 +469,88 @@ default int chronicle()
 		description = ""
 	)
 	void chronicle(int chronicle);
+
+	@ConfigItem(
+		keyName = "showBraceletOfSlaughterCharges",
+		name = "Bracelet of Slaughter Charges",
+		description = "Show Bracelet of Slaughter item charges",
+		position = 26,
+		section = chargesSection
+	)
+	default boolean showBraceletOfSlaughterCharges()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "braceletOfSlaughter",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default int braceletOfSlaughter()
+	{
+		return -1;
+	}
```

Comment:
These should be set using `setRSProfileConfiguration`

Reply:
To be honest, the entire plugin should probably be moved over to use RSProfiles for shared tracking now they exist. I think I'll PR that first instead, then change this PR later after that one has been accepted in some way.